src/Screens/App/index.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+
+import AppLayout from './layout';
+
+class App extends Component {
+  state = {
+    todos: [],
+    input: ''
+  }
+
+  handleAddTodo = () => {
+    this.setState(prevState => ({todos: [...prevState.todos].concat({id: new Date().getTime(), text: prevState.input, checked: false}), input: ''}));
```

Comment:
Shouldn't this `concat` receive an Array ?

Reply:
not neccesary